### DISCUSSION TITLE
Q-Table how to check value between two range

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#9483] [CLI - PWA mode]
Hello friends

i have a custom table which i can input a value between 2 range , i wanted to prompt the user with an error message if it enter another row with a value within a range that already exist . For example Threshold Start contains 1000 and Threshold End contains 2000 .if i add a new Threshold that is between 1000 and 2000 i should be prompt an error message.

The table should look something like this 

![image](https://user-images.githubusercontent.com/84211129/120121855-db286b80-c1b6-11eb-8aea-ee1b6a6fca27.png)
[Table.txt](https://github.com/quasarframework/quasar/files/6566995/Table.txt)

**Comment by metalsadman** [comment]
Add a handler on your q-input's, then you can check there for the values. ie. 
```html
<q-input
                  :value="props.row.ThresholdStart"
                  autofocus
                  mask="#"
                  fill-mask=" "
                  reverse-fill-mask
                  @input="v => { check(v, props.row, 'ThresholdStart', props.row.ThresholdEnd )}"
                ></q-input>
...
```
```js
// handler
check(v, row, colName, tEnd) {
  // pass row object to keep the reference use it like `row[colName]` = v to mutate value
  // condition
  // display error, mutate model etc...
}
...
```

p.s. Please avoid pasting your code snippet as downloadable file, use markdown code syntax instead to display your code https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code.

**Reply by akashramtohul4** [reply]
Helllo metalsadman 

I have try to implement it but its working for some threshold values . am not able to find out the mistake in my logic can please help.

https://codepen.io/akashramtohul4/pen/poeaRgL
